ui/desktop/app/controllers/scopes/scope/projects/settings/index.js

```diff
@@ -4,5 +4,60 @@
  */
 
 import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { getOwner } from '@ember/application';
+import { action } from '@ember/object';
 
-export default class ScopesScopeProjectsSettingsIndexController extends Controller {}
+const THEMES = [
+  {
+    label: 'system',
+    value: 'system-default-theme',
+  },
+  {
+    label: 'light',
+    value: 'light',
+  },
+  {
+    label: 'dark',
+    value: 'dark',
+  },
+];
+
+export default class ScopesScopeProjectsSettingsIndexController extends Controller {
+  @service session;
+  @service ipc;
+  // Methods
+
+  /**
+   * Returns available themes
+   */
+  get themes() {
+    return THEMES;
+  }
+
+  /**
+   * Applies the specified color theme to the root ember element.
+   * @param {string} theme - "light", "dark", or nullish (system default)
+   */
+  @action
+  async toggleTheme({ target: { value: theme } }) {
```

Comment:
Should we be removing the theme selector from the global dropdown now?

Reply:
yup, there was a note on figma that we should remove the global theme selector, I added a separate task for that

Reply:
just to clarify, is this `toggleTheme` function JUST for the theme selector on the global dropdown?

Reply:
🤔 I don't think I follow, can you explain? 

we won't have a theme selector on the global dropdown going forward

Reply:
I was asking if `toggleTheme` functionality does toggle the theme only within the dropdown that is rendered within settings or if is to toggle the theme within the global dropdown?

Reply:
only within the settings page, I have another ticket that gets rid off the global dropdown